docs/Using Fleet/Vulnerability-Processing.md

```diff
@@ -19,91 +13,21 @@ Vulnerable software can have one or more vulnerabilities (CVEs).
 
-We periodically parse the Mac Office release notes posted at
-<https://learn.microsoft.com/en-us/officeupdates/release-notes-office-for-mac> and for each release
-note we extract the version
-information and what vulnerabilities were patched with the release, we then examine all installed
-macOS apps and if an Office app is found we compare its version with the release notes metadata
-and report back any vulnerabilities to which the software is susceptible.
-
-### Linux hosts
-
-First, we determine what Linux distributions are part of your fleet (keep in mind that there will
-be a small delay between the time a new Linux hosts is added and the time the host is 'detected'). We then
-use
-that information to determine what OVAL definitions need to be downloaded and parsed - you can find
-a list of all the OVAL definitions we use
-[here](https://github.com/fleetdm/nvd/blob/master/oval_sources.json). OVAL definitions will be
-refreshed on a daily basis.
-
-Finally, we look at the software inventory of each host and execute the assertions contained in the
-corresponding OVAL file - any match is reported using the same channels as with Windows/Mac OS vulnerabilities
+[Learn more](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Vulnerability-processing.md) about performance and how it works for different platforms.
 
 ## Coverage
 
-For Windows/Mac OS Fleet attempts to detect vulnerabilities for installed software that falls into the following categories (types):
-
-### Apps
-
-- macOS
-- Windows
-
-### Browser plugins
-
-- macOS, Windows
-  - Chrome extensions
-  - Firefox extensions
+Fleet attempts to detect vulnerabilities for installed software that falls into the following categories (types):
 
-### Packages
-
-- macOS
-  - Python
-  - Packages installed using Homebrew
-
-- Windows
-  - Python
-  - Atom
-  - Packages installed using Chocolatey
-
-For Linux, we adhere to whatever is defined in the OVAL definitions, except for:
-
-- Kernel vulnerabilities.
-- Vulnerabilities involving configuration files.
-
-As of right now, the following distributions are supported:
-
-- Ubuntu
-- RHEL based distros (Red Hat, CentOS, Fedora, and Amazon Linux)
-
-As of right now, only app names with all ASCII characters are supported. Apps with names featuring
-non-ASCII characters, such as Cyrillic, will not generate matches.
-
-The ingestion of software varies per platform. For each platform, we run an [osquery query](#ingesting-software-lists-from-hosts) to ingest software.
+| Type                | macOS                                      | Windows                                          | Linux            |
+| ------------------- | ------------------------------------------ | ------------------------------------------------ | ---------------- |
+| Apps                | Yes                                        | Yes                                              | --               |
```

Comment:
Maybe we use the ✅ and ❌ icons instead of "Yes" and "No"?

We use these on the [OS updates doc page](https://fleetdm.com/docs/using-fleet/mdm-macos-updates):
![Screenshot 2023-11-27 at 5 47 30 PM](https://github.com/fleetdm/fleet/assets/47070608/338592f0-6ed6-4e0e-beeb-69dd025ed14c)